Classes/Domain/DataBackend/MySqlDataBackend/MySqlDataBackend.php

```diff
@@ -848,9 +848,27 @@ protected function buildAggregateSQLByConfigCollection(AggregateConfigCollection
 
         /** @var QueryBuilder $queryBuilder */
         $queryBuilder = $connection->createQueryBuilder();
+
+        $queryBuilder->selectLiteral($this->listQueryParts['SELECT'])
+            ->from($this->listQueryParts['FROMTABLE'], $this->listQueryParts['FROMALIAS'] ?? null);
```

Comment:
Die Parameter vom from würde ich in einzelne Zeilen packen, dann ist besser sichtbar, wo das ?? dazugehört